tar_scm.py

```diff
@@ -54,6 +54,10 @@ def run_cmd(cmd, cwd, interactive=False, raisesysexit=False):
     env = os.environ.copy()
     env['LANG'] = 'C'
 
+    if is_proxy_defined():
+        env['http_proxy'] = os.environ.get('http_proxy')
```

Comment:
Is there a reason not to stick to the convention of environment variables being uppercase?